### ISSUE TITLE
Why is so low accuracy in using CIFAR dataset?

### ISSUE THREAD

**Issue by FeiH36** [AshwinRJ/Federated-Learning-PyTorch#31]
The accuracy is about 40% under the following condition. Is there an improving way?
Local epoch:10
batch size: 10
global epoch: 50
learning rate: 0.5 ~ 0.001

**Comment by noobyzy** [comment]
the same for me

**Comment by NSS-01** [comment]
the model is not powerful for cifar-10, you can change other better model for training over the data set
